runtime/src/main/java/org/corfudb/runtime/collections/CorfuStreamEntry.java

```diff
@@ -0,0 +1,60 @@
+package org.corfudb.runtime.collections;
+
+import com.google.protobuf.Message;
+import lombok.Getter;
+
+/**
+ * This will be the type of each entry returned by CorfuStore's Streamer Interface
```

Comment:
I think you can reword the first sentence to:
Entry returned by CorfuStore's Streamer Interface.

Reply:
done.